view/src/components/account.js

```diff
@@ -0,0 +1,332 @@
+// account.js
+
+import React, { Component } from 'react';
+
+import withStyles from '@material-ui/core/styles/withStyles';
+import Typography from '@material-ui/core/Typography';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import { Card, CardActions, CardContent, Divider, Button, Grid, TextField } from '@material-ui/core';
+
+import clsx from 'clsx';
+
+import axios from 'axios';
+import { authMiddleWare } from '../util/auth';
+
+const styles = (theme) => ({
+	content: {
+		flexGrow: 1,
+		padding: theme.spacing(3)
+	},
+	toolbar: theme.mixins.toolbar,
+	root: {},
+	details: {
+		display: 'flex'
+	},
+	avatar: {
+		height: 110,
+		width: 100,
+		flexShrink: 0,
+		flexGrow: 0
+	},
+	locationText: {
+		paddingLeft: '15px'
+	},
+	buttonProperty: {
+		position: 'absolute',
+		top: '50%'
+	},
+	uiProgess: {
+		position: 'fixed',
+		zIndex: '1000',
+		height: '31px',
+		width: '31px',
+		left: '50%',
+		top: '35%'
+	},
+	progess: {
+		position: 'absolute'
+	},
+	uploadButton: {
+		marginLeft: '8px',
+		margin: theme.spacing(1)
+	},
+	customError: {
+		color: 'red',
+		fontSize: '0.8rem',
+		marginTop: 10
+	},
+	submitButton: {
+		marginTop: '10px'
+	}
+});
+
+class account extends Component {
```

Comment:
Same question about class names here.  Is there some reason this must be lowercase?  If not, let's use the normal convention and call it Account.